yargs/index.d.ts

```diff
@@ -187,6 +196,7 @@ declare namespace yargs {
         default?: any;
         defaultDescription?: string;
         demand?: boolean | string;
```

Comment:
I see both usages still listed in the tests: ([`string`](https://github.com/yargs/yargs/blob/v6.6.0/test/usage.js#L768) / [`boolean`](https://github.com/yargs/yargs/blob/v6.6.0/test/usage.js#L1361)) but only the `boolean` usage listed in the docs.